lib/presets.js

```diff
@@ -321,8 +321,48 @@ module.exports = {
           return callback(error);
         }
       });
-    }
+    },
+
+    /**
+     * @summary Update package version in Python Init file
+     * @function
+     * @public
+     *
+     * @param {Object} options - options
+     * @param {String} cwd - current working directory
+     * @param {String} version - version
+     * @param {Function} callback - callback (error)
+     * @returns {null}
+     *
+     * @example
+     * presets.updateVersion.initPy({}, process.cwd(), '1.0.0', (error) => {
+     *   if (error) {
+     *     throw error;
+     *   }
+     * });
+     */
+    initPy: (options, cwd, version, callback) => {
+      _.defaults(options, {
+        targetFile: '__init__.py'
+      });
 
+      const initFile = path.join(cwd, options.targetFile);
+      const cleanedVersion = semver.clean(version);
+
+      if (!cleanedVersion) {
+        return callback(new Error(`Invalid version: ${version}`));
+      }
+
+      replace({
+        files: initFile,
+        from: /(__version__\s*=\s*)'(?:0|[1-9]\d*)\.(?:0|[1-9]\d*)\.(?:0|[1-9]\d*)'/g,
+        to: '$1\'' + cleanedVersion + '\''
```

Comment:
I guess this would break if somebody happened to have `__version__ = "1.0.0"` in their `__init__.py` file? You might be able to allow for both styles of quotes with something like:
```
from: /(__version__\s*=\s*)('|")(?:0|[1-9]\d*)\.(?:0|[1-9]\d*)\.(?:0|[1-9]\d*)\2/g,
to: '$1$2' + cleanedVersion + '$2'
```
?

Reply:
@lurch nice, I've not noticed this 👍

Reply:
@majorz I know nothing about Rust or Cargo files, but do they allow both double-quotes and single-quotes? If they do, perhaps a similar fix to what I've suggested here would also be useful to the Rust support you added to versionist?

Reply:
@lurch I checked the TOML specification and there are single-quote strings as well - literal strings, which cannot be escaped. I will PR this.